src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -731,6 +732,12 @@ export default class ReimbursementRequestService {
 
     if (!reimbursementRequest) throw new NotFoundException('Reimbursement Request', reimbursementRequestId);
 
+    const recipientSettings = await prisma.user_Settings.findUnique({
```

Comment:
move all this code down to line 763, it should hopefully fix all the failing tests